main.js

```diff
@@ -0,0 +1,43 @@
+function generateNumbers() {
+    let numbersOnScreen = Math.floor(Math.random() * 3) + 3;
+    let allNumbers = [];
+
+    for (let i = 0; i < numbersOnScreen; i++) {
+        let randNumber = Math.floor(Math.random() * 101);
+
+        allNumbers.push(randNumber);
+    }
+
+    return allNumbers;
+}
+
+function displayNumbers(allNumbers) {
+    let numbersOnScreen = allNumbers.length;
+
+     // using a for loop instead of forEach as the array.pop changes the length over each iteration
+     for (let i = 0; i < numbersOnScreen; i++) {
+        let randNumber = allNumbers.pop();
+        let number = document.createElement('div');
+
+        number.textContent = randNumber;
+        number.setAttribute('class', 'numberBox');
+
```

Comment:
formatting issues here with whitespace and such/

Reply:
Nah this is fine, in my opinion.